vertx-web-openapi/src/main/java/io/vertx/ext/web/openapi/RouterFactoryOptions.java

```diff
@@ -130,4 +139,36 @@ public RouterFactoryOptions setOperationModelKey(String operationModelKey) {
     this.operationModelKey = operationModelKey;
     return this;
   }
+
+  public RouteNamingStrategy getRouteNamingStrategy() {
+    return routeNamingStrategy;
+  }
+
+  /**
+   * The strategy to follow when naming the generated routes.
+   * @param routeNamingStrategy
+   * @return this object
+   */
+  @Fluent
+  public RouterFactoryOptions setRouteNamingStrategy(RouteNamingStrategy routeNamingStrategy) {
+    this.routeNamingStrategy = routeNamingStrategy;
+    return this;
+  }
+
+
+  public enum RouteNamingStrategy implements Function<Operation, String> {
```

Comment:
Can you add a small javadoc here?

Reply:
Done. Found and fixed some source and doc gen issues along the way.